controllers/usersignup/usersignup_controller_test.go

```diff
@@ -1788,12 +1731,9 @@ func TestUserSignupDeactivatedAfterMURCreated(t *testing.T) {
 				Reason: "DeactivationInProgress",
 			})
 
-		murs := &toolchainv1alpha1.MasterUserRecordList{}
-
 		// The MUR should have now been deleted
-		err = r.Client.List(context.TODO(), murs)
-		require.NoError(t, err)
-		require.Len(t, murs.Items, 0)
+		murtest.AssertThatMasterUserRecords(t, r.Client).HaveCount(0)
+		murtest.AssertThatMasterUserRecord(t, "john-doe", r.Client).DoesNotExist()
```

Comment:
btw, do we still need the:
```go
murtest.AssertThatMasterUserRecord(t, "john-doe", r.Client).DoesNotExist()
```
when we already checked that the list of the MURs is 0?
```suggestion
		murtest.AssertThatMasterUserRecords(t, r.Client).HaveCount(0)
```
the same applies also to all following cases.

Reply:
yeah, you're right. I probably made those latest changes too quickly did not pay attention to these cases 🤷‍♂️

Reply:
done in 12c4184